app/_src/policies/meshcircuitbreaker.md

```diff
@@ -579,3 +579,7 @@ spec:
 We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/{{ page.version }}/reference/http-api).
 {% endtab %}
 {% endtabs %}
+
+## Complete spec
```

Comment:
It's definitely not a blocker for me, but maybe `## Complete specification` would be more appropriate? 🤔

Reply:
Changed it to just be `All policy options` since the output isn't really a spec per se.